index.html

```diff
@@ -9,6 +9,17 @@
     <title>Aziz Mejri - Portfolio</title>
   </head>
   <body>
+    <div class="mobile-menu">
+      <div class="mobile-menu-close">
+        <i class="mobile-menu-close-btn fas fa-times"></i>
+      </div>
+      <ul class="mobile-menu-options">
+        <li><a href="projects">Portoflio</a></li>
```

Comment:
Kindly let's update the `href` of the `Portfolio` option, so that when the user clicks this option it will take the user to the correct part of the page. You can do the same things as you did for the `About` and `Contact` options by referring to them with their `id`.